creator-node/src/routes/nodeSync.js

```diff
@@ -312,9 +312,15 @@ async function _nodesync (req, walletPublicKeys, creatorNodeEndpoint) {
 
     req.logger.info(redisKey, `Successful export from ${creatorNodeEndpoint} for wallets ${walletPublicKeys} and requested min clock ${localMaxClockVal + 1}`)
 
-    // Attempt to connect directly to target CNode's IPFS node.
-    await _initBootstrapAndRefreshPeers(req, body.data.ipfsIDObj.addresses, redisKey)
-    req.logger.info(redisKey, 'IPFS Nodes connected + data export received')
+    try {
+      // Attempt to connect directly to target CNode's IPFS node.
+      await _initBootstrapAndRefreshPeers(req, body.data.ipfsIDObj.addresses, redisKey)
```

Comment:
since this function also has a try-catch and swallows the error, this outer try-catch will never get hit right. instead we would fail -> log error (from line 597) -> come to line 318 and log success which is misleading. i think just the try-catch here is fine and leave the function unchanged

wdyt

Reply:
yeah i was wondering how to structure this but i like this best. i'll do this